net/usrsock/Kconfig

```diff
@@ -76,12 +76,10 @@ config NET_USRSOCK_NPOLLWAITERS
 config NET_USRSOCK_UDP
 	bool "User-space daemon provides UDP sockets"
 	default n
-	select NET_UDP_NO_STACK
```

Comment:
why remove the selection? can we disable the selection in defconfig?

Reply:
@xiaoxiang781216 san,

I want to use Usrsock socket and Kernel socket(such like inet, rndis), and switch it by dynamic.
So, if we do it, we need to remove this one.

Reply:
@SPRESENSE it's better to remove "select NET_UDP_NO_STACK" from NET_USRSOCK_UDP, and change the default value of NET_UDP_NO_STACK like this:
```
config NET_UDP_NO_STACK
	bool "Disable UDP/IP Stack"
	default NET_USRSOCK_UDP if NET_USRSOCK
	select NET_UDP
	---help---
		Build without UDP/IP stack even if UDP networking support enabled.
```
so, we don't need change any exist config and drop the third patch "configs: Update defconfig files".

Reply:
@xiaoxiang781216 san
Thank you for your review feed back and I'm so sorry for late response.
I applied your solution for this PR.
Could you review again?